molecule/docker/destroy.yml

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
Why this file is needed? Why not just link in `molecule.yml` to one from `defaults/` ?